homeassistant/bootstrap.py

```diff
@@ -281,6 +282,12 @@ def mount_local_lib_path(config_dir: str) -> str:
     Async friendly.
     """
     deps_dir = os.path.join(config_dir, 'deps')
-    if deps_dir not in sys.path:
-        sys.path.insert(0, os.path.join(config_dir, 'deps'))
+    env = os.environ.copy()
+    env['PYTHONUSERBASE'] = os.path.abspath(deps_dir)
+    args = [sys.executable, '-m', 'site', '--user-site']
+    process = Popen(args, stdin=PIPE, stdout=PIPE, stderr=PIPE, env=env)
+    stdout, _ = process.communicate()
```

Comment:
Can we use this instead: https://docs.python.org/3/library/asyncio-subprocess.html#create-a-subprocess-low-level-api-using-subprocess-popen